src/tests/ftest/util/server_utils_base.py

```diff
@@ -86,6 +79,11 @@ def get_sub_command_class(self):
             # Use the sub_command parameter from the test's yaml
             sub_command = self.sub_command.value
 
+        # If specified use the configuration file from the YamlParameters object as the default
+        default_config = None
```

Comment:
I am probably missing something, but it seems that it default_config sould be initialized in the following way.
```suggestion
        default_config = DEFAULT_CONFIG_FILE 
```

Reply:
I think in the context of the test the --config option should only be specified if the caller provides it. @phender to confirm

Reply:
The code path ultimately calls `get_default_config_file`, and it looks like `DEFAULT_CONFIG_FILE` isn't used anymore. Probably old code that should be removed (pre-existing)

Reply:
@daltonbohning thanks for the explanation.